app/v2/controllers/V2MovementsController.scala

```diff
@@ -821,139 +829,147 @@ class V2MovementsControllerImpl @Inject() (
   ): Action[UpscanResponse] =
     Action.async(parse.json[UpscanResponse](UpscanResponse.upscanResponseReads)) {
       implicit request =>
-        val originalHc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
-
-        // If the client ID is provided, ensure we add it to the headers.
-        val clientIdHeader = clientId
-          .map[Seq[(String, String)]](
-            id => Seq(XClientIdHeader -> id.value)
-          )
-          .getOrElse(Seq.empty)
+        if (config.enablePhase5) {
```

Comment:
No point doing it here as this is only from Upscan, which will never be triggered.